app/models/protocol.rb

```diff
@@ -288,7 +296,7 @@ def unique_rm_id_to_protocol
     sort_order  = arr[1]
     case sort_name
     when 'id'
-      order("id #{sort_order.upcase}")
+      order("protocols.id #{sort_order.upcase}")
```

Comment:
@kyle-glick @jwiel86 After investigation, looks like this shouldn't actually conflict with Hanna's stuff. Should be fine.